### PR TITLE
[CLOUDGA-23292] : Add cluster-name flag to commands to show up in help

### PR DESCRIPTION
**Pull request by adityakaushik99-yb**
- Closes CLOUDGA-23292
- Tested help command

### CODE REVIEW

**Review by arishta-yb — APPROVED** [review]

File: cmd/cluster/connection-pooling/connection_pooling.go [hunk 1]
```diff
@@ -104,6 +104,10 @@ func performConnectionPoolingOperation(operationName string, cmd *cobra.Command,
 
 func init() {
 	util.AddCommandIfFeatureFlag(ConnectionPoolingCmd, enableConnectionPoolingCmd, util.CONNECTION_POOLING)
+	enableConnectionPoolingCmd.Flags().String("cluster-name", "", "[REQUIRED] The name of the cluster to get details.")
```
Comment (harshitha-yb) [inline cmd/cluster/connection-pooling/connection_pooling.go:107]: why is this required? It is added as part of a global flag. Ref: https://yugabyte.atlassian.net/browse/CLOUDGA-23029

Comment (harshitha-yb) [reply cmd/cluster/connection-pooling/connection_pooling.go:107]: The description is misleading/incorrect too
